components/tensorflow-notebook-image/jupyter_notebook_config.py

```diff
@@ -47,3 +50,21 @@
     # Restrict access to the file
     os.chmod(pem_file, stat.S_IRUSR | stat.S_IWUSR)
     c.NotebookApp.certfile = pem_file
+
+if "S3_ENDPOINT" in os.environ:
+  # Parse Tensorflow-style variables into something we can use.
+  # FIXME I don't see an obvious way to implement S3_VERIFY_SSL, so anything that uses HTTPS has to use proper certs.
+  if os.getenv("S3_USE_HTTPS") != "0":
```

Comment:
this will pass if env is unset or empty str. Add if os.getenv .. and os.getenv != "0"

Reply:
This is just copying the upstream tensorflow way of doing things, wanted to keep behavior consistent.